problemsSpec.js

```diff
@@ -0,0 +1,56 @@
+describe("First Problem", function() {
+	var array=new Array(1,2,3,4);

```

Comment:
You should write the prepare statement (in this case the array creation) in a the test case.